heron/proto/topology.proto

```diff
@@ -69,7 +75,10 @@ message Config {
     required string key = 1;
     // Only one of the below are set
     optional string value = 2;
-    optional bytes java_serialized_value = 3;
+    //optional bytes java_serialized_value = 3;
+    optional bytes serialized_value = 3;
+
+    required ConfigValueType type = 4;
```

Comment:
adding a new required field will make parsing of any existing protos break. Does StreamSchema get persisted anywhere across topology runs where this would break things or is it only used within a running topology?